<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="Minimum_horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="Maximum_Humity_horizontalLayout">
         <item>
          <widget class="QLabel" name="Minimum_label">
           <property name="text">
            <string>Minimum Ground Humity (%):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Minimum_humity_spinBox">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Maximum_Humitylabel">
           <property name="text">
            <string>Maximum Ground Humity: (%)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Maximum_Humity_spinBox">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Update_PushButton">
     <property name="text">
      <string>UPDATE/INITIALIZE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="State_PushButton">
     <property name="text">
      <string>STOP/CONTINUE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="humity_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="estado_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
